build.gradle

```diff
@@ -83,14 +84,9 @@ dependencies {
     scoverage "org.scoverage:scalac-scoverage-plugin_%%:${versions.scalacScoverage}"
     scoverage "org.scoverage:scalac-scoverage-runtime_%%:${versions.scalacScoverage}"
 
-    runtime "com.h2database:h2:${versions.h2}"
-    runtime "mysql:mysql-connector-java:${versions.mysqlConnector}"
-    runtime "org.tinylog:slf4j-binding:${versions.tinylogSlf4jBinding}"
-}
-
-// exclude concrete log bindings from gatling
-configurations {
-    all*.exclude group: 'ch.qos.logback', module:'logback-classic'
+    runtimeOnly "com.h2database:h2:${versions.h2}"
+    runtimeOnly "mysql:mysql-connector-java:${versions.mysqlConnector}"
+    runtimeOnly "org.tinylog:slf4j-binding:${versions.tinylogSlf4jBinding}"
```

Comment:
it appears `runtime` configuration is deprecated in favor of `runtimeOnly` https://docs.gradle.org/4.8.1/userguide/java_library_plugin.html

i'm not sure why `runtime` configuration got promoted to compile scope in the pom though, but this seems to fix it.